File: codingbat/python/src/string-2/end_other.py
def end_other(a, b):
  a = a.lower();
  b = b.lower();
  if len(a) > len(b):
    if a[len(a)-len(b):] == b:
      return True;
  if b[len(b)-len(a):] == a:
    return True;
  return False;

File: codingbat/java/src/string-2/endOther.java
public boolean endOther(String a, String b) {
  a = a.toLowerCase();
  b = b.toLowerCase();
  if(a.length() >= b.length()){
    return a.substring(a.length()-b.length()).equals(b);
  }
  if(b.length() > a.length()){
    return b.substring(b.length()-a.length()).equals(a);
  }
  return false;
}
